pallets/did/src/benchmarking.rs

```diff
@@ -102,6 +102,9 @@ benchmarks! {
 
 		let submitter: AccountIdentifierOf<T> = account(DEFAULT_ACCOUNT_ID, 0, DEFAULT_ACCOUNT_SEED);
 
+		let balance = <CurrencyOf<T> as Currency<AccountIdentifierOf<T>>>::minimum_balance() + <T as Config>::Deposit::get() + <T as Config>::Fee::get();
+		<CurrencyOf<T> as Currency<AccountIdentifierOf<T>>>::make_free_balance_be(&submitter, balance);
```

Comment:
C: Move this to a function for easier readability, e.g., `fn airdrop(who: T::AccountId)`.